### PR TITLE
fix(awsome-notifications): typo for `confirmOk`

### PR DESCRIPTION
**Pull request by andreasbhansen**
The current interface for `AwnLabels` contains a minor typo `confirmOK` which should have been `confirmOk` (see [official docs](https://github.com/f3oall/awesome-notifications/blob/8a1d3104c509964ebd7ff91202b41c936df37f48/docs/docs/customization/labels.md#confirmok)).
Demo: https://codesandbox.io/s/busy-keller-xzkv8f?file=/src/index.js:0-491

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: (https://github.com/f3oall/awesome-notifications/blob/8a1d3104c509964ebd7ff91202b41c936df37f48/docs/docs/customization/labels.md#confirmok

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## awesome-notifications ([<kbd>unpkg</kbd>](https://unpkg.com/browse/awesome-notifications@latest/))
was missing the following properties:
1. AWN
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 03e76583b151ddef31d7ecc46e57edbb0b487043
</p>

File: types/awesome-notifications/awesome-notifications-tests.ts [hunk 1]
```diff
@@ -1,4 +1,12 @@
-import AWN from 'awesome-notifications';
+import AWN from "awesome-notifications";
```
Comment (peterblazejewicz) [inline types/awesome-notifications/awesome-notifications-tests.ts:1]:
```suggestion
import AWN from 'awesome-notifications';
```
DT repo uses prettier global settings, with single quotes

File: types/awesome-notifications/awesome-notifications-tests.ts [hunk 2]
```diff
@@ -1,4 +1,12 @@
...
-awn.alert('Hello world');
+
+const confirmed = () => console.log("Confirmed");
+const cancelled = () => console.log("Cancelled");
+
+awn.confirm(`Tests new confirmOk property name`, confirmed, cancelled, {
+  labels: {
+    confirmOk: "New property name works"
+  }
+});
```
[the remark this answers is not in the document]
Comment (andreasbhansen) [reply types/awesome-notifications/awesome-notifications-tests.ts]: Thanks, Piotr!

**Review by seebeen — APPROVED** [review]

**Comment by seebeen** [comment]
@andreasbhansen Thanks for the improvement. Much appreciated. I've approved your PR.

**Comment by andreasbhansen** [comment]
Ready to merge

**Pull request merged**